cli/tests/test_diff_command.rs

```diff
@@ -43,7 +43,7 @@ fn test_diff_basic() {
 
     let stdout = test_env.jj_cmd_success(&repo_path, &["diff", "-s"]);
     insta::assert_snapshot!(stdout, @r###"
-    R file1
+    D file1
     M file2
     A file3
```

Comment:
I suppose "C" for "created" would be the more natural counterpart to "deleted", but "C" can be interpreted as "copied", so this "A"/"M"/"D" seems like the pragmatic choice.